Performance/PerformanceOld.md

```diff
@@ -1,68 +1,21 @@
-
-## Legacy Fabric 1.8.X
-
-| Name | Known Incompatibilities | Description | Author | Performance Improvement | [Label](/README.md/#labels) | License |
-| --- | :---: | :---: | :---: | :---: | :---: | :---: |
-| [Alternate Current](https://modrinth.com/mod/alternate-current) [![Modrinth Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/Modrinth.png)](https://modrinth.com/mod/alternate-current)[![CurseForge Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/CurseForge.png)](https://www.curseforge.com/minecraft/mc-mods/alternate-current)[![Github Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/Github.png)](https://github.com/SpaceWalkerRS/alternate-current) | Unknown | Alternate Current is an efficient and non-locational redstone dust implementation. | Space Walker | Server | None | [MIT](/licenses/Licenses.md#mit)
-| [LegacyEnhance](https://github.com/Hankung7183/LegacyEnhance) [![Github Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/Github.png)](https://github.com/Hankung7183/LegacyEnhance) | Unknown | A Minecraft mod for LegacyFabric 1.8.9 that enhances performance and fixes bugs. | Hankung7183 | Client | CC BY-NC-SA 4.0 |
-| [Legacy VanillaFix](https://modrinth.com/mod/legacyvanillafix) [![Modrinth Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/Modrinth.png)](https://modrinth.com/mod/legacyvanillafix)[![Github Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/Github.png)](https://github.com/YetAnotherMcModder/Legacy-VanillaFix/) | Unknown | Legacy VanillaFix is a mod that fixes bugs and annoyances in vanilla Minecraft. | piper742 | Both | None | [MIT](/licenses/Licenses.md#mit) |
-| [Mipmaplevel and Language Fix](https://modrinth.com/mod/mipmaplevelandlanguagefix) [![Modrinth Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/Modrinth.png)](https://modrinth.com/mod/mipmaplevelandlanguagefix)[![Github Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/Github.png)](https://github.com/KingContaria/Mipmaplevel-and-Language-Fix) | Unknown | Makes changing Mipmaplevels and Language faster, by instead of doing a full reload, only reloading the respective part of the game. | KingContaria | Client | None | [CC0](/licenses/Licenses.md#cc0)
-| [SaltLib](https://github.com/DeexCoding/SaltLib) [![Github Logo](https://raw.githubusercontent.com/TheUsefulLists/assets/main/Images/Platform_Icons/Github.png)](https://github.com/DeexCoding/SaltLib) | Unknown | SaltLib aims to be an alternative of the Soduim mod for Minecraft, but for legacy versions (1.3.2-1.13.2) of the game. | DeexCoding | Client | None | [Custom](https://github.com/DeexCoding/SaltLib/blob/1.8.9/LICENSE.txt)
-
-## Legacy Fabric 1.7.X
+## Babric b1.7.3
```

Comment:
Babric? You mean Fabric?

Reply:
> Babric? You mean Fabric?

Babric is a fork of Fabric (or Legacy Fabric?) for b1.7.3